custom/icds_reports/migrations/sql_templates/create_datasource_views.sql

```diff
@@ -452,6 +452,59 @@ CREATE VIEW agg_awc_daily_view AS
         ("awc_location"."doc_id" = "agg_awc"."awc_id")
     );
 
+CREATE TABLE IF NOT EXISTS public."config_report_icds-cas_static-child_health_cases_a46c129f"
```

Comment:
Unfortunately, this table is queried in migrations (https://github.com/dimagi/commcare-hq/pull/17565/files/13d993b82de9c42e7b6b86f48b62928d3452ecd5#diff-1af23fff6ceb126cf3ab59253ffb2b7bR542) which means it should be created before migrations (which I think is not possible)

Do you see better solution to this?